package.json

```diff
@@ -51,10 +51,10 @@
     "cypress-keycloak": "^2.0.0",
     "cypress-mochawesome-reporter": "^3.5.1",
     "eslint": "^8.57.0",
-    "typescript-eslint": "^7.12.0",
     "eslint-plugin-cypress": "^2.12.1",
     "eslint-plugin-jest": "^27.1.6",
     "jest": "^29.3.1",
-    "nodemon": "^2.0.20"
+    "nodemon": "^2.0.20",
+    "typescript-eslint": "^7.12.0"
```

Comment:
was there any fixes for the package ? just curious why its changed other than that all looks good to me

Reply:
No packages are the same it's just now alphabetically ordered